ci/scripts/github/common.sh

```diff
@@ -51,12 +58,18 @@ export DISPLAY_ARTIFACT_URL="${DISPLAY_URL}${ARTIFACT_ENDPOINT}"
 # Set sccache env vars
 export SCCACHE_S3_KEY_PREFIX=srf-${NVARCH}-${BUILD_CC}
 export SCCACHE_BUCKET=rapids-sccache
-export SCCACHE_REGION=us-west-2
+export SCCACHE_REGION="${AWS_DEFAULT_REGION}"
 export SCCACHE_IDLE_TIMEOUT=32768
 #export SCCACHE_LOG=debug
 
-gpuci_logger "Environ:"
-env | sort
+mkdir -p ${WORKSPACE_TMP}
+
+function print_env_vars() {
+    gpuci_logger "Environ:"
+    env | grep -v -E "AWS_ACCESS_KEY_ID|AWS_SECRET_ACCESS_KEY|TOKEN" | sort
+}
```

Comment:
Are these not hidden by the Github UI since they are secrets? This makes me nervious

Reply:
Github actions does filter/remove secrets from being printed to the logs, however the docs also say we should avoid printing them in general:
https://docs.github.com/en/actions/security-guides/encrypted-secrets#accessing-your-secrets
> Warning: GitHub automatically redacts secrets printed to the log, but you should avoid printing secrets to the log intentionally.

Which is why I added that in, as an extra just-in-case.

Reply:
Ok, we should then add CONDA_TOKEN to the list of envs not to print

Reply:
Its a regex so it will exclude any env var with "TOKEN" in the name, so this will match on `GH_TOKEN` and `CONDA_TOKEN`

Reply:
Ah I see